react-example/src/index.tsx

```diff
@@ -38,25 +38,28 @@ const HomeLink = styled(Link)`
 `;
 
 ((): void => {
-  const { setEmail } = initialize(process.env.API_KEY || '', ({ email }) => {
-    return axios
-      .post(
-        'http://localhost:5000/generate',
-        {
-          exp_minutes: 2,
-          email,
-          jwt_secret: process.env.JWT_SECRET
-        },
-        {
-          headers: {
-            'Content-Type': 'application/json'
+  const { setUserID: setEmail } = initialize(
```

Comment:
testing code to be reverted back to what it was before the PR is merged - this is just for reviewers' convenience.